tests/std/include/test_header_units_and_modules.hpp

```diff
@@ -545,9 +545,11 @@ void test_print() {
     using namespace std;
     puts("Testing <print>.");
     println("Hello, world!");
+    println();
 

```

Comment:
I figured out what was causing the test coverage to hang in the CI! `P2093R14_formatted_output` does very special things to create a console without popping up windows during a test run, and it interacts with lit in such a way that the test hangs if any assertion failures occur. These asserts are visible if the test is directly compiled and run. (The problems were things like expecting newlines to be embedded in strings, when the `get_console_line` machinery will perceive that line number ending.)

I was able to restore test coverage - some exactly captured from your original commit, other parts reworked.